### PR TITLE
Push javadoc with HTML5.

### PR DESCRIPTION
**Pull request by johnou**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+
+sudo: false
+
 jdk:
-  - oraclejdk8
-  - openjdk8
+- openjdk8
+- openjdk10
+- oraclejdk8
+- oraclejdk10
```
Comment (kuujo) [inline .travis.yml]: Seems like a lot of maybe unnecessary overlap for all those tests

Comment (johnou) [reply .travis.yml]: could perhaps drop oraclejdk then?

**Comment by kuujo** [comment]
I'll let you merge this whenever you want.

**Comment by johnou** [comment]
Let's follow up with openjdk10 failure in another PR.

**Pull request merged**